lib/active_merchant/billing/gateways/adyen.rb

```diff
@@ -554,6 +554,8 @@ def parse(body)
       end
 
       def commit(action, parameters, options)
+        parameters.update(externalPlatform: application_id) if application_id
```

Comment:
I believe `external_platform` needs to be nested within `application_info`. Additionally should we move this to only `purchase` and `authorize` requests based on the [Adyen docs](https://linkprotect.cudasvc.com/url?a=https%3a%2f%2fdocs.adyen.com%2fapi-explorer%2f%23%2fPayment%2fv64%2fpost%2fauthorise__reqParam_applicationInfo&c=E,1,e_kWFeY-FgPx7qduS_4bOFS7PjR5rVBqmA-zYnsRJnDIB4PQiJmWN7EmV_tSxm73BpmSSmf9FF8HskH4e-UTZg7PuoZn2LkJQRCRGR5Q9YMOR7slL-BkLySe&typo=1)?

Reply:
If Adyen accepts it on other requests then no harm no foul

Reply:
updated!